pyiode/reports.pyx

```diff
@@ -10,6 +10,10 @@
 
 from reports cimport B_ReportExec, B_ReportLine
 
+# GDM> I don't really know how report works, so I don't know if a report loads a workspace inside the report
+#      or if you execute a report once a workspace is already loaded. If it is the second option,
+#      maybe make this a method on the Workspace class. Otherwise (or maybe either way?), create a Report class and make
+#      an execute() method on it.
```

Comment:
In the C code, there is a C struct `REPFILE` (see https://github.com/plan-be/iode/blob/master/api/iode.h#L1091) which represents an IODE report and alongside, they are a lot of 'private' and 'public' functions to manipulate this C struct. It would definitly worth to group the C struct `REPFILE` and the related C functions into a C++ class but that's not for tomorrow. 

But yes, a *Report* would be very welcome to cleanup the C code and help maintainers. 
But I don't see any real added value a public Python *Report* class for the users.